include/multipass/virtual_machine.h

```diff
@@ -55,7 +56,8 @@ class VirtualMachine : private DisabledCopyMove
         delayed_shutdown,
         suspending,
         suspended,
-        unknown
+        unknown,
+        unavailable,
```

Comment:
Could you remind me of your intentions for this state? What does it translate to underneath? Are VMs force-stopped? Even suspended ones? If so, why do we need a new state?